test/unit/test_borealis.py

```diff
@@ -0,0 +1,1031 @@
+# Copyright 2019 SuperDARN Canada, University of Saskatchewan
+# Author: Marci Detwiller
+"""
+This test suite is to test the implementation for the following classes:
+    BorealisUtilities
+    BorealisSiteRead
+    BorealisSiteWrite
+    BorealisArrayRead
+    BorealisArrayWrite
+    BorealisRestructureUtilities
+    BorealisConvert
+Support for the following Borealis file types:
+    rawrf
+    antennas_iq
+    bfiq
+    rawacf
+And supports conversion of the following Borealis -> DARN types:
+    bfiq -> iqdat
+    rawacf -> rawacf
+"""
+import bz2
+import collections
+import copy
+import filecmp
+import logging
+import numpy as np
+import os
+import random
+import tables
+import unittest
+
+from collections import OrderedDict
+
+import borealis_rawacf_data_sets
+import borealis_bfiq_data_sets
+import pydarn
+
+pydarn_logger = logging.getLogger('pydarn')
+
+# Site Test files
+borealis_site_bfiq_file = "../test_files/20190909.2200.02.sas.0.bfiq.hdf5.site"
+borealis_site_rawacf_file = "../test_files/20190909.2200.02.sas.0.rawacf.hdf5.site"
+borealis_site_antennas_iq_file = "../test_files/20190909.2200.02.sas.0.antennas_iq.hdf5.site"
+borealis_site_rawrf_file = ""
+
+# Array Test files
+borealis_array_bfiq_file = "../test_files/20190909.2200.02.sas.0.bfiq.hdf5"
+borealis_array_rawacf_file = "../test_files/20190909.2200.02.sas.0.rawacf.hdf5"
+borealis_array_antennas_iq_file = "../test_files/20190909.2200.02.sas.0.antennas_iq.hdf5"
+
+# Problem files TODO
+borealis_site_extra_field_file = ""
+borealis_site_missing_field_file = ""
+borealis_site_incorrect_data_format_file = ""
+
+borealis_array_extra_field_file = ""
+borealis_array_missing_field_file = ""
+borealis_array_incorrect_data_format_file = ""
+borealis_array_num_records_error_file = ""
+borealis_empty_file = "../test_files/empty.rawacf"
+
+
+class TestBorealisFunctions(unittest.TestCase):
+    """
+    Testing class for functions from within borealis.py
+    """
+
+    def test_read_borealis_file_bfiq(self):
+        print('Reading site file as records:')
```

Comment:
All of these print statements are going to cause a problem in the future by causing the maximum log file length to be exceeded.

Reply:
should I remove them all or replace them with something?

Reply:
You can use the pydarn logger as *debug* if you think the information is useful.